adafruit_bitmapsaver.py

```diff
@@ -54,44 +61,50 @@ def _write_dib_header(output_file, width, height):
         output_file.write(b"\x00")
 
 
-def _bytes_per_row(source_width):
+def _bytes_per_row(source_width: int) -> int:
     pixel_bytes = 3 * source_width
     padding_bytes = (4 - (pixel_bytes % 4)) % 4
     return pixel_bytes + padding_bytes
 
 
-def _rotated_height_and_width(pixel_source):
+def _rotated_height_and_width(pixel_source: Union[Bitmap, Display]) -> Tuple[int, int]:
     # flip axis if the display is rotated
     if isinstance(pixel_source, Display) and (pixel_source.rotation % 180 != 0):
-        return (pixel_source.height, pixel_source.width)
-    return (pixel_source.width, pixel_source.height)
+        return pixel_source.height, pixel_source.width
+    return pixel_source.width, pixel_source.height
 
 
-def _rgb565_to_bgr_tuple(color):
-    blue = (color << 3) & 0x00F8  # extract each of the RGB tripple into it's own byte
+def _rgb565_to_bgr_tuple(color: int) -> Tuple[int, int, int]:
+    blue = (color << 3) & 0x00F8  # extract each of the RGB triple into it's own byte
     green = (color >> 3) & 0x00FC
     red = (color >> 8) & 0x00F8
-    return (blue, green, red)
+    return blue, green, red
 
 
 # pylint:disable=too-many-locals
-def _write_pixels(output_file, pixel_source, palette):
+def _write_pixels(
+    output_file: BufferedWriter,
+    pixel_source: Union[Bitmap, Display],
+    palette: Optional[Palette],
+) -> None:
     saving_bitmap = isinstance(pixel_source, Bitmap)
     width, height = _rotated_height_and_width(pixel_source)
     row_buffer = bytearray(_bytes_per_row(width))
     for y in range(height, 0, -1):
         buffer_index = 0
         if saving_bitmap:
+            # pixel_source: Bitmap
             for x in range(width):
-                pixel = pixel_source[x, y - 1]
-                color = palette[pixel]
+                pixel = pixel_source[x, y - 1]  # type: ignore
+                color = palette[pixel]  # type: ignore  # handled by save_pixel's guardians
                 for _ in range(3):
```

Comment:
lets remove the `#type: ignore`'s for now. We haven't added mypy to the CI yet so it's okay if it's still raising errors when run locally. Right now we're primarily focused on the "missing type annotations" error for function arguments and returns. The others we'll get around to when we do add mypy to CI and at that time we'll figure out what configuration we want to use which will guide which things if any need the ignore comments.